frontend/front/src/components/form/HeatPumpAddressField/HeatPumpAddressField.jsx

```diff
@@ -0,0 +1,157 @@
+import { Controller, useController } from 'react-hook-form';
```

Comment:
I think this should be also delete since you have moved it into surveypage folder

Reply:
:+1: I think that was just some git nonsense